config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java

```diff
@@ -86,6 +86,7 @@
         @ModelFeatureFlag(owners = {"baldersheim"}) default boolean distributeExternalRankExpressions() { return false; }
         @ModelFeatureFlag(owners = {"baldersheim"}) default int maxConcurrentMergesPerNode() { throw new UnsupportedOperationException("TODO specify default value"); }
         @ModelFeatureFlag(owners = {"baldersheim"}) default int maxMergeQueueSize() { throw new UnsupportedOperationException("TODO specify default value"); }
+        @ModelFeatureFlag(owners = {"baldersheim"}) default boolean dryRunOnnxOnSetup() { throw new UnsupportedOperationException("TODO specify default value"); }
```

Comment:
why do flags have a default implementation if the best (?) way to implement it is to make sure it is not called?

Reply:
Because this is an api between the configmodels and the configserver.
Once use of the method is gone there still must be a default implemenation until the last version is gone.